content/concepts/introduction/protocols/ping.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Ping"
+description: "The ping protocol is a simple request response protocol."
+weight: 20
+---
+
+## What is Ping?
```

Comment:
I don't think we need 3 paragraphs about ICMP ping. I'd prefer to have a single sentence in the description of libp2p ping:
> The libp2p ping protocol is different from the ping command line utility (ICMP ping), as it requires an already established libp2p connection.

Reply:
As an introduction to ping it seems useful to have the additional context.

Reply:
Yeah but this is confusion. We can't have an article about libp2p ping start with 3 paragraphs about something completely different that happens to have the same name.

Reply:
Updated